python_modules/libraries/dagster-tableau/dagster_tableau/translator.py

```diff
@@ -0,0 +1,64 @@
+from enum import Enum
+from typing import Any, Dict
+
+from dagster import _check as check
+from dagster._core.definitions.asset_key import AssetKey
+from dagster._core.definitions.asset_spec import AssetSpec
+from dagster._record import record
+
+
+class TableauContentType(Enum):
+    """Enum representing each object in Tableau's ontology."""
+
+    WORKBOOK = "workbook"
+    VIEW = "view"
+    DATA_SOURCE = "data_source"
+
+
+@record
+class TableauContentData:
+    """A record representing a piece of content in Tableau.
+    Includes the content's type and data as returned from the API.
+    """
+
+    content_type: TableauContentType
+    properties: Dict[str, Any]
```

Comment:
We generally use `Mapping` instead of `Dict`, and `object` instead of `Any`.

Reply:
Oh my bad - I missed this one after your comment in #24233 !

Reply:
Fixed in [380fd2a](https://github.com/dagster-io/dagster/pull/24190/commits/380fd2a270b406ce5937c310f24505a2c50ac929)